Source_Code/TM_UnitTests/TeamMentor.UnitTests/TeamMentor.UserData/Users/Test_Users_Creation.cs

```diff
@@ -157,6 +157,93 @@ public void CreateTmUserSigupResponse_Success()
         }
 
         [Test]
+        public void CreateTmUserSigupResponse_TLD_Email_Success()
+        {
+            UserGroup.None.assert();                //  change current thread privildges to None
+
+            var newUser = new NewUser()
+            {
+                Company = 10.randomLetters(),
+                Country = 10.randomLetters(),
+                Firstname = 10.randomLetters(),
+                Lastname = 10.randomLetters(),
+                Note = 10.randomLetters(),
+                Password = "!r4J" + 10.randomLetters(),
+                State = 10.randomLetters(),
+                Title = 10.randomLetters(),
+                Username = 10.randomLetters(),
+                Email = "dmin@xn--xn--fsqu00a.xn--g6w251d"
+            };

```

Comment:
Is there an method we can call that will return a temp user like this?

ideally this should be changed to 

```csharp
var newUser = new NewUser().random_Data().email("dmin@xn--xn--fsqu00a.xn--g6w251d");
```

or even better

```
var newUser = "dmin@xn--xn--fsqu00a.xn--g6w251d".newUser_Obj_With_Email(); // or something similar
```